src/workflows/index.js

```diff
@@ -1,10 +1,10 @@
-import CodeSaturn   from './code-saturn';
+// import CodeSaturn   from './code-saturn';
 import PyFr         from './pyfr/pyfr-simput';
 import PyFrExec     from './pyfr/pyfr-exec';
 import Visualizer   from './visualizer';
 
 const Workflows = {
-  CodeSaturn,
+  // CodeSaturn,
   PyFr,
   PyFrExec,
   Visualizer,
```

Comment:
It just contains an introduction, nothing we can re-add in a few minutes, removing it.